saleor/graphql/product/schema.py

```diff
@@ -231,8 +231,12 @@ class ProductQueries(graphene.ObjectType):
     def resolve_attributes(self, info, **kwargs):
         return resolve_attributes(info, **kwargs)
 
-    def resolve_attribute(self, info, id):
-        return graphene.Node.get_node_from_global_id(info, id, Attribute)
+    def resolve_attribute(self, info, id=None, slug=None):
+        validate_one_of_args_is_in_query("id", id, "slug", slug)
+        if id:
+            return graphene.Node.get_node_from_global_id(info, id, Product)
+        if slug:
```

Comment:
IMO the second if is no needed. You validated the args in 235, which means that you have id or slug.
```
if id:
    return graphene.Node.get_node_from_global_id(info, id, Product)
return resolve_attribute_by_slug(slug=slug)
```

Reply:
@korycins You're right, but this solution is consistent with our codebase, we have the same structure if few more resolvers. So I think this is fine for this PR.

Reply:
> IMO the second if is no needed. You validated the args in 235, which means that you have id or slug.
> 
> ```
> if id:
>     return graphene.Node.get_node_from_global_id(info, id, Product)
> return resolve_attribute_by_slug(slug=slug)
> ```

I borrowed this from the other resolvers for consistency

Reply:
I'll make it like this
```
if id:
    return graphene.Node.get_node_from_global_id(info, id, Product)
return resolve_attribute_by_slug(slug=slug)
```
but for all the codebase, soon

Reply:
👍 


> @korycins You're right, but this solution is consistent with our codebase, we have the same structure if few more resolvers. So I think this is fine for this PR.

Sounds good 👍

Reply:
@korycins, @maarcingebala Can anyone review the current commit?